src/components/ArrowToggle/ArrowToggle.scss

```diff
@@ -7,6 +7,10 @@ $block: '.#{variables.$ns}arrow-toggle';
     vertical-align: middle;
     transition: transform 0.1s ease-out;
 
+    &_no-transition {
+        transition: transform 0s;
+    }
+
     &_direction {
```

Comment:
IMHO, it can be done with css, for example 
```css
    &_direction {
        &_bottom {
            transform: matrix(1, 0, 0, 1, 0, 0);
        }

        &_left {
            transform: matrix(0, 1, -1, 0, 0, 0);
        }

        &_top {
            transform: matrix(-1, 0, 0, -1, 0, 0);
        }

        &_right {
            transform: matrix(0, -1, 1, 0, 0, 0);
        }
    }
```

Reply:
that's genius, thank you